css/properties.json

```
@@ -9878,15 +9878,15 @@
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-emphasis-color"
   },
   "text-emphasis-position": {
-    "syntax": "[ over | under ] && [ right | left ]",
+    "syntax": "auto | [ over | under ] && [ right | left ]",
```

also, specifying right or left is optional?
```suggestion
    "syntax": "auto | [ over | under ] && [ right | left ]?",
```